### PR TITLE
Enable Disable and other

### PR DESCRIPTION
**Pull request by beetofly**
**Version update to v2.0.0 - for a pre_release**

Changes:
* Adds Disable/Enable functionality to leuchtfeuers housekeeping
* Update namespaces to LeuchtfeuerHousekeepingBundle
* Update command name to `Leuchtfeuer:housekeeping`
* Update Readme

Test these changes, please.

### CODE REVIEW

File: Command/EventLogCleanupCommand.php [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 class EventLogCleanupCommand extends Command
 {
-    protected static $defaultName = 'mautic:leuchtfeuer:housekeeping';
+    protected static $defaultName = 'Leuchtfeuer:housekeeping';
```
Comment (Moongazer) [inline Command/EventLogCleanupCommand.php:16]: IMO the command name should be lower-case

File: Config/config.php [hunk 2]
```diff
@@ -3,23 +3,50 @@
...
     'name'        => 'Housekeeping by Leuchtfeuer',
     'description' => 'Database Cleanup Command to delete lead_event_log table entries, campaign_lead_event_log table entries, email_stats table entries where the referenced email entry is currently not published and email_stats_devices table entries.',
-    'version'     => '1.4.0',
+    'version'     => '1.4.3',
     'author'      => 'Leuchtfeuer Digital Marketing GmbH',
     'services'    => [
+        'integrations' => [
+            'mautic.integration.housekeepingleuchtfeuer' => [
+                'class' => \MauticPlugin\LeuchtfeuerHousekeepingBundle\Integration\HousekeepingLeuchtfeuerIntegration::class,
+                'tags'  => [
```
Comment (Moongazer) [inline Config/config.php:12]: To keep a streamlined naming convention, the class-name should be `LeuchtfeuerHousekeepingIntegration`

File: Integration/HousekeepingLeuchtfeuerIntegration.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+use Mautic\IntegrationsBundle\Integration\BasicIntegration;
+use Mautic\IntegrationsBundle\Integration\ConfigurationTrait;
+use Mautic\IntegrationsBundle\Integration\Interfaces\BasicInterface;
+
+class HousekeepingLeuchtfeuerIntegration extends BasicIntegration implements BasicInterface
+{
+    use ConfigurationTrait;
+
+    public const INTEGRATION_NAME = 'housekeepingleuchtfeuer';
```
Comment (Moongazer) [inline Integration/HousekeepingLeuchtfeuerIntegration.php:15]: See above (or is the namespace wrong? I'd expect `leuchtfeuerhousekeeping` here as name)

File: README.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ Command to delete lead_event_log table entries, campaign_lead_event_log table en
 Important: If referenced email is ever switched back to published, the contacts will get the email again.
 
 ```
-bin/console mautic:leuchtfeuer:housekeeping
+bin/console Leuchtfeuer:housekeeping
```
Comment (Moongazer) [inline README.md:10]: Lowercase command name

**Review by Moongazer — COMMENTED** [review]
I'd suggest to keep a clean naming convention.

File: Command/EventLogCleanupCommand.php (hunk 1 above)
Comment (beetofly) [reply Command/EventLogCleanupCommand.php]: Changed to lower case with commit 3979ef0

File: README.md (hunk 4 above)
Comment (beetofly) [reply README.md]: Changed to lower case with commit 3979ef0

File: Config/config.php (hunk 2 above)
Comment (beetofly) [reply Config/config.php:12]: I tried it already before with `LeuchtfeuerHousekeepingIntegration` (and once again after your comment), but although I changed 'INTEGRATION_NAME' to 'leuchtfeuerhousekeeping' and all config entries to 'leuchtfeuerhousekeeping' the `Enable/Disable` switch does not work. Thatswhy I integrated the solution in my first commit with the class-name 'HouskeepingLeuchtfeuerIntegration' (the Enable/Disbale Switch is shown and after injecting at the right place also working). 

So I finally kept the class-name 'HouskeepingLeuchtfeuerIntegration' with my current commit and did not apply your suggestion (sorry).

File: Integration/HousekeepingLeuchtfeuerIntegration.php (hunk 3 above)
Comment (beetofly) [reply Integration/HousekeepingLeuchtfeuerIntegration.php:15]: See my comment before.

File: Config/config.php (hunk 2 above)
Comment (Moongazer) [reply Config/config.php:12]: OK thats strange

**Review by Moongazer — APPROVED** [review]

**Review by oltmanns-leuchtfeuer — APPROVED** [review]
Tested and LGMT

**Pull request merged**